doc/pre_install.rst

```diff
@@ -0,0 +1,70 @@
+.. include:: links.inc
+
+.. _preinstall:
+
+Before you install
+==================
+
+.. contents::
+   :local:
+   :depth: 1
+
+Overview of the MNE tools suite
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The project started with what is now called :doc:`MNE-C <manual/c_reference>` —
+a set of interrelated command-line and GUI programs focused on computing
+cortically constrained Minimum Norm Estimates from MEG and EEG data. These
+tools were written in C by Matti Hämäläinen.
+
+:ref:`MNE-python <api_reference>` reimplements the functionality of MNE-C,
+and extends considerably the analysis and visualization capabilities.
+MNE-python is collaboratively developed and has more than 150 contributors.
+
+The :ref:`ch_matlab` provides a MATLAB interface to the .fif file format and
+other MNE data structures, and provides example MATLAB implementations of
+some of the core analysis functionality of MNE-C. It is distributed alongside
+MNE-C, and can also be downloaded from the `MNE-MATLAB git repository`_.
+
+:ref:`mne_cpp` provides core MNE functionality implemented in C++ and is
+primarily intended for embedded and real-time applications.
+
+There are also python tools for easily importing MEG data from the Human
+Connectome Project for use with MNE-python (`MNE-HCP`_), and tools for
+managing MNE projects so that they comply with the Brain Imaging Data Structure
+specification (`MNE-BIDS`_).
+
+What should I install?
+^^^^^^^^^^^^^^^^^^^^^^
+
+If you intend only to perform ERP, ERF, or other sensor-level analyses,
+:doc:`MNE-python <install_mne_python>` is all you need. If you
+prefer MATLAB over python, probably all you need is :doc:`MNE-C
+<install_mne_c>` — the MNE MATLAB toolbox is distributed with it.
```

Comment:
Somehow you should note that MNE-C+MNE-MATLAB has fewer features than MNE-Python.